### ISSUE TITLE
missing :

### ISSUE THREAD

**Issue by DragonXexe** [nanobyte-dev/nanobyte_os#18]
in src/main.asm I think there is a : missing on line 57

**Comment by chibicitiberiu** [comment]
Can you specify where exactly? There is no `main.asm` in the latest version of the repository.

**Comment by ajh123** [comment]
He might be using an older commit and/or a different branch/repository.

**Comment by aeonrex** [comment]
In tag Part1 of the "Building an OS" tutorial. Make throws a warning on macOS.

<img width="725" alt="image" src="https://user-images.githubusercontent.com/9206148/222938170-e5849fce-051a-4399-b23a-d2d37b665283.png">

**Comment by ajh123** [comment]
After looking at it myself I think the line should be:
```nasm
.halt:
```
and not
```nasm
.halt
```

**Comment by chibicitiberiu** [comment]
Yes, that is correct. Unfortunately I can't modify it because it is an old commit.

**Comment by aeonrex** [comment]
Great, thanks for the info.
